bedrock/firefox/templates/firefox/whatsnew/beta/whatsnew-fx68.html

```diff
@@ -2,32 +2,82 @@
  # License, v. 2.0. If a copy of the MPL was not distributed with this
  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
 
-{% add_lang_files "firefox/whatsnew_66" %}
```

Comment:
Wait, I thought we said we we're going to remove these templates for beta (like you have for dev edition), since changing the lang file could cause problems? Shouldn't beta fallback to the default /whatsnew?

Reply:
You're right, I need to do beta as well. Sorry this has been so messy, gonna try to clean it up.